rudder-agent-thin/SOURCES/patches/rudder-agent-thin/0001-rudder-agent-to-rudder-agent-thin-debian.patch

```diff
@@ -109,34 +109,34 @@ diff -Naur a/rules b/rules
 --- a/rules	2014-12-08 15:05:20.562188524 +0100
 +++ b/rules	2014-12-08 15:09:17.030590768 +0100
 @@ -15,8 +15,6 @@
- 	dh_testdir
- 	# dh_prep has been moved from 'install' target since we need to build LMDB first
- 	dh_prep
```

Comment:
You reindent a patch
This context is reindented, which means the original file has been reindented too, but I can't find where